docs/chapter4-payload-handoff-format.rst

```diff
@@ -1052,6 +1052,15 @@ skipped if unsupported by the platform.
                       by OS runtime services.
     runtime-data      Runtime service data memory region which will be used
                       by OS runtime services.
+    soft-reserve      Specific-purpose memory (SPM). The memory is earmarked
```

Comment:
soft-reserve == special-purpose to align with UEFI. We could mention additionally that CXL/HBM are examples. thx!

Reply:
which one we would like to use in UPL spec? soft-reserve or special-purpose?

Reply:
done.